opensrp-chw/src/main/assets/json.form/child_enrollment.json

```diff
@@ -60,15 +60,15 @@
         "openmrs_entity": "",
```

Comment:
Dont overwrite files, create news ones and and prefix them with **tz** like tz_child_enrollment.json